pkg/sac/testutils/search_test_cases.go

```diff
@@ -0,0 +1,221 @@
+			"process indicators": {
+			ScopeKey: Cluster1NamespacesBCReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster1: {
+					testconsts.NamespaceB: 3,
+					testconsts.NamespaceC: 3,
+				},
+			},
+		},
+		"Cluster2 read-write access should only see Cluster2 process indicators": {
+			ScopeKey: Cluster2ReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceA: 3,
+					testconsts.NamespaceB: 3,
+					testconsts.NamespaceC: 3,
+				},
+			},
+		},
+		"Cluster2 and NamespaceA read-write access should see Cluster2 and NamespaceA process indicators": {
+			ScopeKey: Cluster2NamespaceAReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceA: 3,
+				},
+			},
+		},
+		"Cluster2 and NamespaceB read-write access should only see Cluster2 and NamespaceB process indicators": {
+			ScopeKey: Cluster2NamespaceBReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceB: 3,
+				},
+			},
+		},
+		"Cluster2 and NamespaceC read-write access should only see Cluster2 and NamespaceC process indicators": {
+			ScopeKey: Cluster2NamespaceCReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceC: 3,
+				},
+			},
+		},
+		"Cluster2 and Namespaces A and B read-write access should only see appropriate cluster/namespace " +
+			"process indicators": {
+			ScopeKey: Cluster2NamespacesABReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceA: 3,
+					testconsts.NamespaceB: 3,
+				},
+			},
+		},
+		"Cluster2 and Namespaces A and C read-write access should only see appropriate cluster/namespace " +
+			"process indicators": {
+			ScopeKey: Cluster2NamespacesACReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceA: 3,
+					testconsts.NamespaceC: 3,
+				},
+			},
+		},
+		"Cluster2 and Namespaces B and C read-write access should only see appropriate cluster/namespace " +
+			"process indicators": {
+			ScopeKey: Cluster2NamespacesBCReadWriteCtx,
+			Results: map[string]map[string]int{
+				testconsts.Cluster2: {
+					testconsts.NamespaceB: 3,
+					testconsts.NamespaceC: 3,
+				},
+			},
+		},
+	}
+}
+
+// GenericUnrestrictedSACSearchTestCases returns a generic set of SACSearchTestCase.
+// It is appropriate to use when the store contains 27 objects.
+func GenericUnrestrictedSACSearchTestCases(_ *testing.T) map[string]SACSearchTestCase {
+	return map[string]SACSearchTestCase{
+		"global read access should see all process indicators": {
+			ScopeKey: UnrestrictedReadCtx,
+			Results: map[string]map[string]int{
+				"": {"": 27},
+			},
+		},
+		"global read-write access should see all process indicators": {
+			ScopeKey: UnrestrictedReadCtx,
```

Comment:
UnrestrictedReadWriteCtx ? (same applies to GenericUnrestrictedRawSACSearchTestCases)